.ci.yaml

```diff
@@ -1849,6 +1848,7 @@ targets:
 
   - name: Linux deferred components
     recipe: flutter/deferred_components
+    presubmit: true
```

Comment:
This is not needed: presubmit is defaulted as true.
(it has been running in presubmit: https://ci.chromium.org/p/flutter/builders/try/Linux%20deferred%20components)

Reply:
Ok, I'll remove it!